runelite-client/src/main/java/net/runelite/client/plugins/info/InfoPanel.java

```diff
@@ -170,12 +186,79 @@ void init()
 			}
 		});
 
+		JLabel pluginLabel = new JLabel();
+		pluginLabel.setForeground(ColorScheme.LIGHT_GRAY_COLOR);
+		pluginLabel.setFont(smallFont);
+		pluginLabel.setText("Plugins");
+
+		JRichTextPane restoreLabel = new JRichTextPane();
+		restoreLabel.setForeground(Color.WHITE);
+		restoreLabel.setFont(smallFont);
+		restoreLabel.enableAutoLinkHandler(false);
+		restoreLabel.addHyperlinkListener(e ->
+		{
+			if (HyperlinkEvent.EventType.ACTIVATED.equals(e.getEventType()))
+			{
+				final int result = JOptionPane.showOptionDialog(null,
+							"This will disable all plugins except the default ones.",
+							"Are you sure?", JOptionPane.YES_NO_OPTION, JOptionPane.WARNING_MESSAGE,
+							null, new String[]{"Yes", "No"}, "No");
+
+				if (result == JOptionPane.YES_OPTION)
+				{
+					for (Plugin plugin : pluginManager.getPlugins())
+					{
+						PluginDescriptor pluginDescriptor = plugin.getClass().getAnnotation(PluginDescriptor.class);
+						boolean stopPlugin = !pluginDescriptor.enabledByDefault();
+						ExternalPluginManifest manifest = ExternalPluginManager.getExternalPluginManifest(plugin.getClass());
+						if (manifest != null)
+						{
+							if (externalPluginManager.getInstalledExternalPlugins().contains(manifest.getInternalName()))
+							{
+								stopPlugin = true;
+							}
+						}
+						if (stopPlugin)
+						{
+							try
+							{
+								pluginManager.setPluginEnabled(plugin, false);
+								pluginManager.stopPlugin(plugin);
+							}
+							catch (PluginInstantiationException ex)
+							{
+								log.warn("Unable to stop {}", plugin.getClass().getSimpleName());
```

Comment:
This should log the exception as well. See [line 324 of PluginListPanel](https://github.com/runelite/runelite/blob/78eb69c8df8f823a590fe267bf9663591c3c09c0/runelite-client/src/main/java/net/runelite/client/plugins/config/PluginListPanel.java#L324):

```java
log.warn("Error when stopping plugin {}", plugin.getClass().getSimpleName(), ex);
```